collectors/misc/score/src/main/java/com/capitalone/dashboard/repository/ScoreCollectorRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.capitalone.dashboard.repository;
```

Comment:
@sidbha007  do we need this class

Reply:
yes, i think it is needed by CollectorTask in getCollectorRepository.